source/aws-connect-vm-serverless/src/main/java/com/amazonaws/kvstream/KVSProcessRecordingLambda.java

```diff
@@ -65,14 +73,32 @@ private boolean processData(String data) {
             return false;
         }
 
+        AmazonDynamoDBClientBuilder builder = AmazonDynamoDBClientBuilder.standard();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code is written so that the client cannot be reused across invocations of the Lambda function.
To improve the performance of the Lambda function, consider using static initialization/constructor, global/static variables and singletons. It allows to keep alive and reuse HTTP connections that were established during a previous invocation.
Learn more about [best practices for working with AWS Lambda functions](https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html).